lib/RESTClient.js

```diff
@@ -412,8 +417,12 @@ class RESTClient {
                 ret.push(data);
                 retLen += data.length;
             }).on('error', callback).on('end', () => {
-                this.endRespond(res, Buffer.concat(ret, retLen).toString(),
-                    log, callback);
+                if (req.url === '/_/healthcheck') {
+                    callback(null, res);
```

Comment:
I didn't use it because I specifically need the response object to be returned. Would it be better to move the conditional into the endRespond function? I was trying to minimize function calls :)